frontend/README.md

```diff
@@ -29,11 +29,10 @@ You can use the `Dapp` component as a starting point for your project. It has
 comments explaining each part of its code, and indicating what's specific to
 this project, and what can be reused.
 
-## Feedback, help and news
+## Getting help and news about Hardhat
```

Comment:
```suggestion
## Getting help and news
```
Alternatively: "Getting help with and news about Hardhat". But that's a mouthful. The problem with the way it's written is that it implies "Getting help about Hardhat" and "Getting news about Hardhat", but you don't "get help _about_" something, you "get help _with_" something.